src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -11671,7 +11668,7 @@ public void Main(string s, string? ns)
             var listNS = List.Create(ns);
             listS /*T:List<string!>!*/ .ToString();
             listNS /*T:List<string?>!*/ .ToString();
-            listS = listNS!; // warn 1
+            listS = listNS!; // 1
```

Comment:
What does the `1` signify now that we are no longer warning here?

Reply:
The `// 1` here just helps distinguish lines that look very similar and easily check that they don't appear in the diagnostics.

---
In reply to: [245121556](https://github.com/dotnet/roslyn/pull/32090#discussion_r245121556) [](ancestors = 245121556)